<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadImagewindow</class>
 <widget class="QMainWindow" name="UploadImagewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upload Image</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/Logos/output-onlinepngtools (2).png</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Logo/Background.png);</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">#Logo{
background-color: transparent;
border-image: url(:/Logo/Logo.png);
background: none;
border: none;
background-repeat: none;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UploadButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#UploadButton{
background-color: transparent;
border-image: url(:/Logo/UploadCar_up.png);
background: none;
border: none;
background-repeat: none;
}
#UploadButton:pressed
{
   border-image: url(:/Logo/UploadCar_down.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="DamageclassificationButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#DamageclassificationButton{
background-color: transparent;
border-image: url(:/Logo/GetDmgClass_up.png);
background: none;
border: none;
background-repeat: none;
}
#DamageclassificationButton:pressed
{
   border-image: url(:/Logo/GetDmgClass_down.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#BackButton{
background-color: transparent;
border-image: url(:/Logo/Back_up.png);
background: none;
border: none;
background-repeat: none;
}
#BackButton:pressed
{
   border-image: url(:/Logo/Back_down.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{
background-color: transparent;
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilenameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Logotext">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Brush Script MT</family>
      <pointsize>28</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#Logotext{
background-color: transparent;
border-image: url(:/Logo/Logo1.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Logotext_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Brush Script MT</family>
      <pointsize>28</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#Logotext_2{
background-color: transparent;
border-image: url(:/Logo/Logo2.png);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HelpButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click here for help&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#HelpButton{
background-color: transparent;
border-image: url(:/Logo/Help_up.png);
background: none;
border: none;
background-repeat: none;
}
#HelpButton:pressed
{
   border-image: url(:/Logo/Help_down.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>336</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Gui-pngs/Logos.qrc"/>
 </resources>
 <connections/>
</ui>
